build.rs

```diff
@@ -149,16 +178,28 @@ fn build(frameworks_path: &str) {
         .expect("could not write bindings");
 }
 
-#[cfg(any(target_os = "macos", target_os = "ios"))]
 fn main() {
-    if let Ok(directory) = frameworks_path() {
-        build(&directory);
+    let (target, platform) = match std::env::var("TARGET") {
```

Comment:
Isn't there a better way of doing this? All of this should be exposed via cfg, no?

Reply:
I believe cfg will return the building platform as this is build.rs, so here it would always be darwin. Unless I'm wrong. I can't check right now unfortunately.